client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -588,25 +582,41 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	renderSiteLaunchedTask = ( TaskComponent, baseProps, task ) => {
-		const { needsVerification, translate } = this.props;
+		const { needsVerification, siteIsUnlaunched, translate } = this.props;
 		const disabled = ! baseProps.completed && needsVerification;
 
 		return (
 			<TaskComponent
 				{ ...baseProps }
 				bannerImageSrc="/calypso/images/stats/tasks/launch.svg"
 				buttonText={ translate( 'Launch site' ) }
-				completedTitle={ translate( 'You launched your site' ) }
-				description={ translate(
-					"Your site is private and only visible to you. When you're ready, launch your site to make it public."
-				) }
+				completedButtonText={ siteIsUnlaunched && translate( 'Launch site' ) }
+				completedTitle={
+					siteIsUnlaunched
+						? translate( 'You skipped launching your site' )
+						: translate( 'You launched your site' )
+				}
+				description={
+					siteIsUnlaunched
+						? translate(
+								"Your site is private and only visible to you. When you're ready, launch your site to make it public."
+						  )
+						: /*
+						   * This string should not actually show.
+						   * If the site is launched, the `completedButtonText` will be false above.
+						   * That should prevent the task from being expanded.
+						   * Let's go ahead and include it anyway just in case so we aren't incorrectly telling people their site is unlaunched
+						   */
+						  translate( 'Congratulations! Your site is visible to the world.' )
```

Comment:
I put this in as a "just in case" so if the checklist task display logic ever changes, we don't end up incorrectly telling someone their site is private.  Happy to take it out if someone can think of a better way to handle this.